src/components/InputPanel/index.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import './styles.css';
+
+// Libraries
```

Comment:
These comments are irrelevant

Reply:
I like to organize the imports that way

Reply:
Why?

Reply:
I like to separate the components with the libraries, helper functions, stylesheets etc..

Reply:
That way it is easier to see what is imported and whether a import is used or not

Reply:
That's redundant:
- Components: Always in `components` folders
- Libraries: Recognisable by the non-relative path
- Stylesheets: They are always `.css` files
- Helper functions: All other imports

Also, let ESLint warn you when an import isn't used! That's what it's made for :)

Reply:
I will take a look at other projects how they are solving this.